website/docs/guides/debug-schema-names.md

```diff
@@ -93,7 +93,11 @@ Now, re-read through the logic of your `generate_schema_name` macro, and mentall
 
 You should find that the schema dbt is constructing for your model matches the output of your `generate_schema_name` macro.
 
-Be careful. Snapshots do not follow this behavior, check out the docs on [target_schema](/reference/resource-configs/target_schema) instead.
+<VersionBlock lastVersion="1.8">
+
+Be careful. Snapshots do not follow this behavior, check out the docs on [target_schema](/reference/resource-configs/target_schema) instead. 
```

Comment:
Since these configs will still exist, I think it's good to leave this warning but rephrase. Something like:

> Be careful. Snapshots do not follow this behavior if [target_schema](/reference/resource-configs/target_schema) is set. To have environment-aware snapshots, remove the `target_schema` config from your snapshots. If you still want a custom schema for your snapshots, use the `schema` config instead.